lib/localeapp/updater.rb

```diff
@@ -31,9 +31,9 @@ def update(data)
       end
     end
 
-    def dump(data)
+    def dump(data, target_dir = Localeapp.configuration.translation_data_directory)
```

Comment:
Since we use global state for the configuration anyway, cannot we set
the default directory in `Localeapp::CLI::Pull#execute`, make this new
parameter for `#dump` mandatory, and then refactor duplicated code in
`Localeapp::CLI::Pull#update_backend`?